File: Linked List/203. Remove Linked List Elements.py
class Solution:
    def removeElements(self, head, value):

        if head is None:
            return head

        while head is not None and head.val == value:
            head = head.next

        temp = head

        while temp is not None and temp.next is not None:

            if temp.next.val == value:
                temp.next = temp.next.next
            else:
                temp = temp.next

        return head

File: Linked List/203. Remove Linked List Elements.cpp
class Solution {
public:
    ListNode* removeElements(ListNode* head, int value) {
        if(head == NULL)
            return head;

        while(head != NULL && head->val == value)
        {
            head = head->next;
        }
        ListNode* temp = head;
        while(temp != NULL && temp->next != NULL)
        {
            if(temp->next->val == value)
                temp->next = temp->next->next;
            else
                temp = temp->next;
        }
        return head;
    }
};
